SonarQube.TeamBuild.PreProcessor/SonarLintAnalyzerProvider.cs

```diff
@@ -82,16 +84,21 @@ private static bool IsCSharpPluginInstalled(ISonarQubeServer server)
         private static bool TryCreateRuleset(ISonarQubeServer server, TeamBuildSettings settings, string projectKey, ILogger logger)
         {
             logger.LogDebug(Resources.SLAP_FetchingSonarLintRuleset);
-            string content = TryGetProfileExportForProject(server, projectKey, logger);
 
-            if (content != null)
+            string rulesetContent;
```

Comment:
I believe we're spelling "ruleset" in code as ruleSet / RuleSet to be consistent with how Roslyn spells it